packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -476,6 +476,25 @@ export const messages = {
   Workflow: 'Workflows',
   XOrgHub: 'X Org Hubs',
 
+  conflict_detect_error:
+    'An error was encountered during conflict detection. %s',
+  conflict_detect_retrieve_org_source:
+    'Conflict Detection: retrieving org source',
+  conflict_detect_convert_org_source:
+    'Conflict Detection: converting org source',
+  conflict_detect_conflicts_during_deploy:
+    'Conflicts were detected during the metadata deploy',
```

Comment:
Small thing but do you think the modal should say something about the conflicting files being shown in the output view?